ONSdigital/ssdc-rm-caseprocessor
# SDCSRM-373 Improve Python Implementation of Case Ref

# Motivation and Context
In a previous spike, some changes were proposed to make the code for the Format Preserving Encryption more production suitable.

# What has changed
Changes and reasoning documented in [write up](https://github.com/ONSdigital/ssdc-rm-documentation/pull/238)
Added tests where the expected results are copied from the Java version

# How to test?
Check the write up looks ok
Run the tests by in python_spike dir running `make test`
Run against the ATs and check the case refs look ok

# Links
[Trello](https://trello.com/c/gJE87a45)
[Jira](https://jira.ons.gov.uk/browse/SDCSRM-373)

# Screenshots (if appropriate):
